package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@google-cloud/speech",
   "description": "Cloud Speech Client Library for Node.js",
-  "version": "1.0.1",
+  "version": "1.0.2",
```

Comment:
This change is more of a feature than a bug fix, so I think 1.1.0 might be more appropriate.

Reply:
Agreed - changing to 1.1.0.